### PR TITLE
chore(nucleus): remove flapping downstreams

### PR DESCRIPTION
**Pull request by nolanlawson**
This downstream has flapped twice recently. I've filed bugs on the team. Let's remove it for now.

### CODE REVIEW

**Comment by nolanlawson** [comment]
luvio is also flapping, I filed a bug on them a month ago, let's remove it

**Comment by nolanlawson** [comment]
lwr-recipes also flapped, I will file a bug on them and remove it as well

File: .nucleus.yaml [hunk 1]
```diff
@@ -16,10 +16,7 @@ branches:
         - communities/ui-feeds-components
         - communities/ui-lightning-community
         - lwc/lwc-platform
-        - salesforce/lds-lightning-platform
```
Comment (ravijayaramappa) [inline .nucleus.yaml:19]: You could comment them out with a reference to the bug.

**Review by ravijayaramappa — APPROVED** [review]

File: .nucleus.yaml (hunk 1 above)
Comment (nolanlawson) [reply .nucleus.yaml:19]: Good idea. How about I start a Quip doc instead? I need to keep track of all of our downstreams that we're deliberately blocklisting.

**Pull request merged**